src/redux/reducers/ui.js

```diff
@@ -24,12 +25,14 @@ export default handleActions(
           isModalOpen: true,
           ...payload,
         },
+        chatBubble: 'minimised',
       };
     },
-    [actionTypes.UI_CLOSE_MODAL](state) {
+    [actionTypes.UI_CLOSE_MODAL](state, { payload }) {
       return {
         ...state,
         modalProps: defaultState.modalProps,
+        chatBubble: payload ? payload.chatBubble : 'bubble',
```

Comment:
Why are you mixing modal reducers and chat bubble?

Reply:
@akshatnitd This is done because in full screen we don't want the chat bubble. Also chat bubble shall not be present whenever a dialog opens up. So its is placed inside the ui reducer.

Reply:
For that the zIndex of Modal should be higher than Chat bubble or dispatch another action to toggle bubble state.

Reply:
@akshatnitd Made the changes.